aws/components/datafeed/state.ftl

```diff
@@ -38,13 +38,13 @@
                     },
                     "backuplgstream" : {
                         "Id" : formatDependentResourceId(AWS_CLOUDWATCH_LOG_GROUP_STREAM_RESOURCE_TYPE, lgId, "backup"),
-                        "Name" : "S3Delivery",
+                        "Name" : "Backup",
```

Comment:
I think the names listed are the ones used by Firehose. This name is used on the streams and I don't think they can be changed

Reply:
OK - I'll revert them...